recipes/pywin32/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pywin32" %}
+{% set version = "220" %}
+{% set hash_type = "sha1" %}
+{% set hash = "88ba6a31c077a9e1b49b59dc215ddadc6aa5e3d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/Build {{ version }}/{{ name }}-{{ version }}.zip
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: 
+    - python setup.py install --record=record.txt  # [not py3k]
+    - python setup3.py install --record=record.txt  # [py3k]
+  skip: True  # [not win]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
```

Comment:
Oh that is very interesting. My understanding was that you always need those features if your package has C extensions, but what you say makes perfectly sense if Python already declares and tracks the appropriate runtime.